doc/QUICK_START.md

```diff
@@ -92,14 +92,21 @@ These are generic (and condensed) installation instructions for the **current de
 
         foreman start -f Procfile.dev
 
+   The application will be available at `http://127.0.0.1:8080`. To customize
```

Comment:
The default seems to be port 5000 for me...

Looking at the procfile that confused me too... Don't know where the 5000 originates...